### PR TITLE
Set android app SHA fingerprint.

### PR DESCRIPTION
**Pull request by mjovkovic**

### CODE REVIEW

File: lib/server/server.js [hunk 1]
```diff
@@ -416,6 +419,73 @@ function configureApp(vetId, platform) {
...
+      } else if (apps.length === 1) {
+        let shaCert = projectManagement.shaCertificate(fingerprint.replace(/:/g, ''));
+
+        apps[0].addShaCertificate(shaCert)
+          .then(response => {
+            resolve(response);
+          })
+          .catch(error => {
+            if (error && error.errorInfo && error.errorInfo.code === 'project-management/already-exists') {
+              resolve('Requested entity already exists');
```
Comment (mslobodan) [inline lib/server/server.js:476]: @mjovkovic Can we read existing fingerprint, or check if it exists before trying to add fingerprint?

Comment (mjovkovic) [reply lib/server/server.js:476]: @mslobodan we could, but what would we accomplish with that? We get that info if we try to upload it again :D

Comment (mslobodan) [reply lib/server/server.js:476]: @mjovkovic OK, I thought if someone already uploaded wrong one, and with next build we want to fix it. But that will never happen as we are setting it through build script. 😀

Comment (mjovkovic) [reply lib/server/server.js:476]: 👍

**Pull request merged**